fuzz/src/lib.rs

```diff
@@ -48,11 +48,31 @@ pub enum Action<O: Debug> {
 	IterNext,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct Layer<V> {
 	// The stored value per possible key (depends if we have ref counting or not)
 	pub values: [Option<V>; NUMBER_OF_POSSIBLE_KEYS],
-	pub written: bool,
+	pub written: WrittenState,
+}
+
+impl<V: Debug> Debug for Layer<V> {
+	fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+		let mut output = f.debug_map();
+		output.entry(&"written", &self.written);
+		for (k, v) in self.values.iter().enumerate() {
+			if let Some(v) = v {
+				output.entry(&k, v);
+			}
+		}
+		output.finish()
+	}
+}
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum WrittenState {
```

Comment:
Even if it is clear from code, a comment on the difference between Yes and Processed would be good from my point of view.

Reply:
Done